### PR TITLE
77103 | Fix Customizer Display

### PR DESCRIPTION
**Pull request by ggwicz**
_Ref:_ [C#77103](https://central.tri.be/issues/77103)

### CODE REVIEW

File: image-widget.php [hunk 1]
```diff
@@ -47,9 +47,9 @@ public function __construct() {
 			require_once( 'lib/ImageWidgetDeprecated.php' );
 			new ImageWidgetDeprecated( $this );
 		} else {
-			add_action( 'sidebar_admin_setup', array( $this, 'admin_setup' ) );
+			add_action( 'admin_enqueue_scripts', array( $this, 'admin_setup' ) );
 		}
-		add_action( 'admin_head-widgets.php', array( $this, 'admin_head' ) );
+		//add_action( 'admin_head-widgets.php', array( $this, 'admin_head' ) );
```
Comment (zbtirrell) [inline image-widget.php]: delete this line

**Comment by ggwicz** [comment]
@zbtirrell done; ready for re-review

File: image-widget.php [hunk 2]
```diff
@@ -47,9 +47,8 @@ public function __construct() {
 			require_once( 'lib/ImageWidgetDeprecated.php' );
 			new ImageWidgetDeprecated( $this );
 		} else {
-			add_action( 'sidebar_admin_setup', array( $this, 'admin_setup' ) );
+			add_action( 'admin_enqueue_scripts', array( $this, 'admin_setup' ) );
```
Comment (zbtirrell) [inline image-widget.php]: won't this cause our ThickBox styles to override all the admin thick boxes?

Comment (ggwicz) [inline image-widget.php]: can you explain, @zbtirrell ?

the file loaded here only loads this CSS which doesn't affect thickboxes -- unless im missing something: https://github.com/moderntribe/image-widget/pull/36/files#diff-69ba9c14e6b289482864cca019c8fdf4

**Comment by ggwicz** [comment]
@zbtirrell fixed what was reported in Slack — limited the admin.css loading to only widgets page and Customizer pane

**Comment by zbtirrell** [comment]
👍

**Pull request merged**